WORKSPACE

```diff
@@ -100,10 +110,10 @@ git_repository(
 # Google Protocol Buffers.
 http_archive(
     name = "com_google_protobuf",
-    sha256 = "98e615d592d237f94db8bf033fba78cd404d979b0b70351a9e5aaff725398357",
-    strip_prefix = "protobuf-3.9.1",
+    sha256 = "e8c7601439dbd4489fe5069c33d374804990a56c2f710e00227ee5d8fd650e67",
+    strip_prefix = "protobuf-3.11.2",
```

Comment:
Could you also update the version in Dockerfile?

Reply:
Done